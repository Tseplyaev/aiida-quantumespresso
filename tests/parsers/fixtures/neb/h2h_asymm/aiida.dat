      0.0000000000      0.0000000000      2.1764284244
      0.2031850961     -0.1091969895      0.1551318421
      0.4548586820     -0.2649038139      0.2109559635
      0.7190031858     -0.1285280101      0.1251482747
      0.8651377199     -0.1017044313      0.0363979751
      1.0000000000     -0.0000000000      2.1764263761
