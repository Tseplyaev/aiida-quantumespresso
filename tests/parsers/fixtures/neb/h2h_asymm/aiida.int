      0.0000000000      0.0000000000
      0.0040000000     -0.0001124719
      0.0080000000     -0.0004442523
      0.0120000000     -0.0009868888
      0.0160000000     -0.0017319285
      0.0200000000     -0.0026709190
      0.0240000000     -0.0037954074
      0.0280000000     -0.0050969413
      0.0320000000     -0.0065670678
      0.0360000000     -0.0081973345
      0.0400000000     -0.0099792885
      0.0440000000     -0.0119044774
      0.0480000000     -0.0139644483
      0.0520000000     -0.0161507488
      0.0560000000     -0.0184549261
      0.0600000000     -0.0208685276
      0.0640000000     -0.0233831006
      0.0680000000     -0.0259901926
      0.0720000000     -0.0286813507
      0.0760000000     -0.0314481225
      0.0800000000     -0.0342820552
      0.0840000000     -0.0371746962
      0.0880000000     -0.0401175929
      0.0920000000     -0.0431022925
      0.0960000000     -0.0461203426
      0.1000000000     -0.0491632903
      0.1040000000     -0.0522226831
      0.1080000000     -0.0552900682
      0.1120000000     -0.0583569932
      0.1160000000     -0.0614150053
      0.1200000000     -0.0644556518
      0.1240000000     -0.0674704801
      0.1280000000     -0.0704510376
      0.1320000000     -0.0733888717
      0.1360000000     -0.0762755296
      0.1400000000     -0.0791025587
      0.1440000000     -0.0818615064
      0.1480000000     -0.0845439200
      0.1520000000     -0.0871413469
      0.1560000000     -0.0896453344
      0.1600000000     -0.0920474299
      0.1640000000     -0.0943391808
      0.1680000000     -0.0965121343
      0.1720000000     -0.0985578379
      0.1760000000     -0.1004678388
      0.1800000000     -0.1022336845
      0.1840000000     -0.1038469223
      0.1880000000     -0.1052990995
      0.1920000000     -0.1065817635
      0.1960000000     -0.1076864617
      0.2000000000     -0.1086047414
      0.2040000000     -0.1093345419
      0.2080000000     -0.1100728695
      0.2120000000     -0.1109144143
      0.2160000000     -0.1118568661
      0.2200000000     -0.1128979145
      0.2240000000     -0.1140352494
      0.2280000000     -0.1152665605
      0.2320000000     -0.1165895375
      0.2360000000     -0.1180018702
      0.2400000000     -0.1195012483
      0.2440000000     -0.1210853616
      0.2480000000     -0.1227518998
      0.2520000000     -0.1244985527
      0.2560000000     -0.1263230100
      0.2600000000     -0.1282229615
      0.2640000000     -0.1301960968
      0.2680000000     -0.1322401059
      0.2720000000     -0.1343526784
      0.2760000000     -0.1365315040
      0.2800000000     -0.1387742725
      0.2840000000     -0.1410786737
      0.2880000000     -0.1434423973
      0.2920000000     -0.1458631331
      0.2960000000     -0.1483385707
      0.3000000000     -0.1508664001
      0.3040000000     -0.1534443108
      0.3080000000     -0.1560699926
      0.3120000000     -0.1587411353
      0.3160000000     -0.1614554287
      0.3200000000     -0.1642105625
      0.3240000000     -0.1670042264
      0.3280000000     -0.1698341102
      0.3320000000     -0.1726979036
      0.3360000000     -0.1755932964
      0.3400000000     -0.1785179783
      0.3440000000     -0.1814696391
      0.3480000000     -0.1844459686
      0.3520000000     -0.1874446564
      0.3560000000     -0.1904633923
      0.3600000000     -0.1934998661
      0.3640000000     -0.1965517675
      0.3680000000     -0.1996167863
      0.3720000000     -0.2026926122
      0.3760000000     -0.2057769349
      0.3800000000     -0.2088674443
      0.3840000000     -0.2119618300
      0.3880000000     -0.2150577818
      0.3920000000     -0.2181529895
      0.3960000000     -0.2212451428
      0.4000000000     -0.2243319314
      0.4040000000     -0.2274110451
      0.4080000000     -0.2304801736
      0.4120000000     -0.2335370068
      0.4160000000     -0.2365792343
      0.4200000000     -0.2396045459
      0.4240000000     -0.2426106313
      0.4280000000     -0.2455951803
      0.4320000000     -0.2485558826
      0.4360000000     -0.2514904281
      0.4400000000     -0.2543965063
      0.4440000000     -0.2572718071
      0.4480000000     -0.2601140203
      0.4520000000     -0.2629208355
      0.4560000000     -0.2656789766
      0.4600000000     -0.2681991867
      0.4640000000     -0.2704192692
      0.4680000000     -0.2723466525
      0.4720000000     -0.2739887649
      0.4760000000     -0.2753530347
      0.4800000000     -0.2764468905
      0.4840000000     -0.2772777606
      0.4880000000     -0.2778530733
      0.4920000000     -0.2781802570
      0.4960000000     -0.2782667402
      0.5000000000     -0.2781199511
      0.5040000000     -0.2777473182
      0.5080000000     -0.2771562699
      0.5120000000     -0.2763542345
      0.5160000000     -0.2753486404
      0.5200000000     -0.2741469161
      0.5240000000     -0.2727564898
      0.5280000000     -0.2711847899
      0.5320000000     -0.2694392449
      0.5360000000     -0.2675272832
      0.5400000000     -0.2654563330
      0.5440000000     -0.2632338228
      0.5480000000     -0.2608671810
      0.5520000000     -0.2583638359
      0.5560000000     -0.2557312159
      0.5600000000     -0.2529767495
      0.5640000000     -0.2501078649
      0.5680000000     -0.2471319906
      0.5720000000     -0.2440565549
      0.5760000000     -0.2408889863
      0.5800000000     -0.2376367130
      0.5840000000     -0.2343071636
      0.5880000000     -0.2309077663
      0.5920000000     -0.2274459496
      0.5960000000     -0.2239291418
      0.6000000000     -0.2203647713
      0.6040000000     -0.2167602665
      0.6080000000     -0.2131230558
      0.6120000000     -0.2094605676
      0.6160000000     -0.2057802301
      0.6200000000     -0.2020894719
      0.6240000000     -0.1983957213
      0.6280000000     -0.1947064067
      0.6320000000     -0.1910289564
      0.6360000000     -0.1873707988
      0.6400000000     -0.1837393624
      0.6440000000     -0.1801420754
      0.6480000000     -0.1765863664
      0.6520000000     -0.1730796636
      0.6560000000     -0.1696293954
      0.6600000000     -0.1662429903
      0.6640000000     -0.1629278766
      0.6680000000     -0.1596914826
      0.6720000000     -0.1565412368
      0.6760000000     -0.1534845676
      0.6800000000     -0.1505289033
      0.6840000000     -0.1476816722
      0.6880000000     -0.1449503029
      0.6920000000     -0.1423422236
      0.6960000000     -0.1398648628
      0.7000000000     -0.1375256488
      0.7040000000     -0.1353320101
      0.7080000000     -0.1332913749
      0.7120000000     -0.1314111716
      0.7160000000     -0.1296988288
      0.7200000000     -0.1281578237
      0.7240000000     -0.1267059532
      0.7280000000     -0.1253070983
      0.7320000000     -0.1239602051
      0.7360000000     -0.1226642195
      0.7400000000     -0.1214180878
      0.7440000000     -0.1202207557
      0.7480000000     -0.1190711695
      0.7520000000     -0.1179682751
      0.7560000000     -0.1169110185
      0.7600000000     -0.1158983458
      0.7640000000     -0.1149292030
      0.7680000000     -0.1140025361
      0.7720000000     -0.1131172913
      0.7760000000     -0.1122724144
      0.7800000000     -0.1114668515
      0.7840000000     -0.1106995488
      0.7880000000     -0.1099694520
      0.7920000000     -0.1092755075
      0.7960000000     -0.1086166610
      0.8000000000     -0.1079918588
      0.8040000000     -0.1074000467
      0.8080000000     -0.1068401709
      0.8120000000     -0.1063111774
      0.8160000000     -0.1058120121
      0.8200000000     -0.1053416212
      0.8240000000     -0.1048989506
      0.8280000000     -0.1044829465
      0.8320000000     -0.1040925547
      0.8360000000     -0.1037267214
      0.8400000000     -0.1033843926
      0.8440000000     -0.1030645142
      0.8480000000     -0.1027660324
      0.8520000000     -0.1024878932
      0.8560000000     -0.1022290426
      0.8600000000     -0.1019884266
      0.8640000000     -0.1017649913
      0.8680000000     -0.1014245328
      0.8720000000     -0.1006157011
      0.8760000000     -0.0993477627
      0.8800000000     -0.0976514498
      0.8840000000     -0.0955574947
      0.8880000000     -0.0930966297
      0.8920000000     -0.0902995870
      0.8960000000     -0.0871970990
      0.9000000000     -0.0838198978
      0.9040000000     -0.0801987158
      0.9080000000     -0.0763642852
      0.9120000000     -0.0723473384
      0.9160000000     -0.0681786075
      0.9200000000     -0.0638888248
      0.9240000000     -0.0595087227
      0.9280000000     -0.0550690334
      0.9320000000     -0.0506004891
      0.9360000000     -0.0461338221
      0.9400000000     -0.0416997647
      0.9440000000     -0.0373290492
      0.9480000000     -0.0330524079
      0.9520000000     -0.0289005729
      0.9560000000     -0.0249042767
      0.9600000000     -0.0210942513
      0.9640000000     -0.0175012292
      0.9680000000     -0.0141559426
      0.9720000000     -0.0110891238
      0.9760000000     -0.0083315050
      0.9800000000     -0.0059138185
      0.9840000000     -0.0038667966
      0.9880000000     -0.0022211715
      0.9920000000     -0.0010076755
      0.9960000000     -0.0002570409
      1.0000000000     -0.0000000000
