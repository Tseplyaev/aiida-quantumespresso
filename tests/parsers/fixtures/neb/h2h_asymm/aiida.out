
     Program NEB v.6.4.1 starts on 30Jul2019 at 16:40:34 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors

     MPI processes distributed on     1 nodes

     No input file found, assuming nothing to parse
     Searching argument -input_images or --input_images
     Reading input from pw_1.in
     Reading input from pw_2.in

     initial path length           =  4.2553 bohr
     initial inter-image distance  =  0.8511 bohr

     string_method                 =    neb
     restart_mode                  =    from_scratch
     opt_scheme                    =    broyden
     num_of_images                 =    6
     nstep_path                    =    20
     CI_scheme                     =    manual
     first_last_opt                =    F
     use_freezing                  =    F
     ds                            =    2.0000 a.u.
     k_max                         =    0.3000 a.u.
     k_min                         =    0.2000 a.u.
     suggested k_max               =    0.1542 a.u.
     suggested k_min               =    0.1028 a.u.
     path_thr                      =    0.0500 eV / A

     list of climbing images :   4,

     ------------------------------ iteration   1 ------------------------------

     tcpu =      0.2    self-consistency for image   1
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      0.9    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      1.6    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      2.6    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      3.6    self-consistency for image   5
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      4.4    self-consistency for image   6
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   2.306556 eV
     activation energy (<-) =   2.306556 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -43.4377479            3.221834            F
         3        -42.3177121            2.745045            F
         4        -42.3177121            3.469716            F
         5        -43.4377479            3.221825            F
         6        -44.6242679            2.176426            T

     path length          =  4.255 bohr
     inter-image distance =  0.851 bohr

     ------------------------------ iteration   2 ------------------------------

     tcpu =      5.1    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      6.0    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      6.9    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      7.8    self-consistency for image   5
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   1.920208 eV
     activation energy (<-) =   1.920208 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -43.7883672            2.392173            F
         3        -42.7611424            2.681757            F
         4        -42.7040599            3.339607            F
         5        -43.7883654            2.781284            F
         6        -44.6242679            2.176426            T

     path length          =  4.331 bohr
     inter-image distance =  0.866 bohr

     ------------------------------ iteration   3 ------------------------------

     tcpu =      8.7    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      9.5    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     10.4    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     11.2    self-consistency for image   5
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   1.403015 eV
     activation energy (<-) =   1.403015 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.2345566            2.580978            F
         3        -43.3552230            2.833255            F
         4        -43.2212527            3.117258            F
         5        -44.1738716            2.114035            F
         6        -44.6242679            2.176426            T

     path length          =  4.548 bohr
     inter-image distance =  0.910 bohr

     ------------------------------ iteration   4 ------------------------------

     tcpu =     12.0    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     12.8    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     13.7    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     14.5    self-consistency for image   5
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.832743 eV
     activation energy (<-) =   0.832743 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.6036962            2.366008            F
         3        -43.9577499            2.827383            F
         4        -43.7915251            2.694042            F
         5        -44.5128104            1.964249            F
         6        -44.6242679            2.176426            T

     path length          =  4.867 bohr
     inter-image distance =  0.973 bohr

     ------------------------------ iteration   5 ------------------------------

     tcpu =     15.3    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     16.0    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     16.7    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     17.7    self-consistency for image   5
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.274360 eV
     activation energy (<-) =   0.274360 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7372813            2.610411            F
         3        -44.4905688            2.551591            F
         4        -44.3499083            2.175912            F
         5        -44.7274207            1.503494            F
         6        -44.6242679            2.176426            T

     path length          =  5.261 bohr
     inter-image distance =  1.052 bohr

     ------------------------------ iteration   6 ------------------------------

     tcpu =     18.5    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     19.1    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     19.8    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     20.7    self-consistency for image   5
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.144849 eV
     activation energy (<-) =   0.144849 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.4794191            5.781473            F
         3        -44.8282872            2.556546            F
         4        -44.8055464            1.454015            F
         5        -44.6771844            3.670033            F
         6        -44.6242679            2.176426            T

     path length          =  5.699 bohr
     inter-image distance =  1.140 bohr

     ------------------------------ iteration   7 ------------------------------

     tcpu =     21.4    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     22.1    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     22.8    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     23.7    self-consistency for image   5
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.102390 eV
     activation energy (<-) =   0.102390 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7361168            2.694008            F
         3        -44.6255068            1.983803            F
         4        -44.5218777            2.050344            F
         5        -44.7288447            1.345482            F
         6        -44.6242679            2.176426            T

     path length          =  5.266 bohr
     inter-image distance =  1.053 bohr

     ------------------------------ iteration   8 ------------------------------

     tcpu =     24.4    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     25.1    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     25.9    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     26.8    self-consistency for image   5
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.6847978            1.479192            F
         3        -44.7637784            2.178809            F
         4        -44.7440174            1.882355            F
         5        -44.7309914            1.666981            F
         6        -44.6242679            2.176426            T

     path length          =  5.381 bohr
     inter-image distance =  1.076 bohr

     ------------------------------ iteration   9 ------------------------------

     tcpu =     27.5    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     28.1    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     28.9    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     29.9    self-consistency for image   5
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7230862            0.477431            F
         3        -44.7516327            1.927040            F
         4        -44.8698931            0.701726            F
         5        -44.7292303            1.468692            F
         6        -44.6242679            2.176426            T

     path length          =  5.594 bohr
     inter-image distance =  1.119 bohr

     ------------------------------ iteration  10 ------------------------------

     tcpu =     30.6    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     31.3    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     32.1    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     32.6    self-consistency for image   5
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7228744            0.535612            F
         3        -44.7725484            0.395437            F
         4        -44.8671727            0.751173            F
         5        -44.7323493            0.163552            F
         6        -44.6242679            2.176426            T

     path length          =  5.553 bohr
     inter-image distance =  1.111 bohr

     ------------------------------ iteration  11 ------------------------------

     tcpu =     33.1    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     33.8    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     34.4    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     34.9    self-consistency for image   5
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7219047            0.857054            F
         3        -44.7816555            0.329368            F
         4        -44.8714912            0.692837            F
         5        -44.7350002            0.807315            F
         6        -44.6242679            2.176426            T

     path length          =  5.586 bohr
     inter-image distance =  1.117 bohr

     ------------------------------ iteration  12 ------------------------------

     tcpu =     35.5    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     36.1    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     36.8    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     37.4    self-consistency for image   5
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7277469            0.075750            F
         3        -44.7808975            0.478425            F
         4        -44.8798451            0.375046            F
         5        -44.7331474            0.062474            F
         6        -44.6242679            2.176426            T

     path length          =  5.618 bohr
     inter-image distance =  1.124 bohr

     ------------------------------ iteration  13 ------------------------------

     tcpu =     38.0    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     38.4    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     39.0    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     39.6    self-consistency for image   5
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7276582            0.230066            F
         3        -44.7865860            0.164786            F
         4        -44.8783708            0.292777            F
         5        -44.7329451            0.025835            F
         6        -44.6242679            2.176426            T

     path length          =  5.658 bohr
     inter-image distance =  1.132 bohr

     ------------------------------ iteration  14 ------------------------------

     tcpu =     40.1    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     40.8    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     41.4    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     42.1    self-consistency for image   5
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7279994            0.044107            F
         3        -44.7891669            0.263794            F
         4        -44.8724405            0.343625            F
         5        -44.7326820            0.102635            F
         6        -44.6242679            2.176426            T

     path length          =  5.654 bohr
     inter-image distance =  1.131 bohr

     ------------------------------ iteration  15 ------------------------------

     tcpu =     42.6    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     43.1    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     43.9    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     44.8    self-consistency for image   5
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7241904            0.604323            F
         3        -44.8371158            0.528976            F
         4        -44.8029146            0.280008            F
         5        -44.7290787            0.693873            F
         6        -44.6242679            2.176426            T

     path length          =  5.598 bohr
     inter-image distance =  1.120 bohr

     ------------------------------ iteration  16 ------------------------------

     tcpu =     45.5    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     46.1    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     46.8    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     47.3    self-consistency for image   5
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7292809            0.216374            F
         3        -44.8277756            0.105436            F
         4        -44.8034381            0.290112            F
         5        -44.7285167            0.150804            F
         6        -44.6242679            2.176426            T

     path length          =  5.592 bohr
     inter-image distance =  1.118 bohr

     ------------------------------ iteration  17 ------------------------------

     tcpu =     47.8    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     48.3    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     49.0    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     49.8    self-consistency for image   5
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7306645            0.096519            F
         3        -44.8492483            0.429866            F
         4        -44.7741590            0.205397            F
         5        -44.7268583            0.135767            F
         6        -44.6242679            2.176426            T

     path length          =  5.606 bohr
     inter-image distance =  1.121 bohr

     ------------------------------ iteration  18 ------------------------------

     tcpu =     50.4    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     51.1    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     51.9    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     52.6    self-consistency for image   5
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7319221            0.417725            F
         3        -44.8827673            0.352816            F
         4        -44.7522665            0.121389            F
         5        -44.7258329            0.130028            F
         6        -44.6242679            2.176426            T

     path length          =  5.647 bohr
     inter-image distance =  1.129 bohr

     ------------------------------ iteration  19 ------------------------------

     tcpu =     53.2    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     53.8    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     54.3    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     54.8    self-consistency for image   5
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7331582            0.101901            F
         3        -44.8853782            0.340896            F
         4        -44.7534274            0.126827            F
         5        -44.7259027            0.071354            F
         6        -44.6242679            2.176426            T

     path length          =  5.631 bohr
     inter-image distance =  1.126 bohr

     broyden uphill step : history is reset


     ------------------------------ iteration  20 ------------------------------

     tcpu =     55.3    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     55.8    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     56.4    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     57.0    self-consistency for image   5
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7334649            0.155132            F
         3        -44.8891717            0.210956            F
         4        -44.7527959            0.125148            F
         5        -44.7259723            0.036398            F
         6        -44.6242679            2.176426            T

     path length          =  5.640 bohr
     inter-image distance =  1.128 bohr

     ---------------------------------------------------------------------------


     neb: reached the maximum number of steps

     NEB          :     53.68s CPU     57.47s WALL


   This run was terminated on:  16:41:32  30Jul2019            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
